Add option to opt out from WooCommerce Blocks patterns (#9807)
woocommerce/woocommerce-blocks: docs/third-party-developers/extensibility/editor/curating-editor-experience.md

# Curating the Editor Experience

## Disable pattern directory**

> To remove patterns provided by WooCommerce Blocks from being accessed in the Inserter, call `remove_theme_support( 'woocommerce-blocks-patterns' );` as a callback to `after_setup_theme` action in your functions.php file. Follow the example:

Reviewer: Related to the conversation with @nefeline elsewhere, I think it'd be helpful to steer potential assumptions away from this being a theme-only API.
```suggestion
To remove patterns provided by WooCommerce Blocks from being accessed in the Inserter, call `remove_theme_support( 'woocommerce-blocks-patterns' );` in a callback on the `after_setup_theme` action. For example:
```

# Curating the Editor Experience

## Disable pattern directory**

To remove patterns provided by WooCommerce Blocks from being accessed in the Inserter, the following can be added to your functions.php file:

> `remove_theme_support( 'woocommerce-blocks-patterns' );`

Reviewer 2: After the change we need to update the doc to say they need to call this after `after_setup_theme` action hook.
